index.html

```diff
@@ -185,6 +185,23 @@
     </section>
     <section>
       <h2>
+        Permission description
+      </h2>
+      <dl class='idl' title='dictionary PermissionDescriptor'>
+        <dt>
+          required PermissionName <a>name</a>
+        </dt>
+      </dl>
+      <p>
+        A permission is defined by a name and other properties that depends on
+        the name. The simplest permissions require only a name but some others
+        will require more information. In which case, they should have an
+        <dfn>associated PermissionDescriptor</dfn> dictionary that inherits
```

Comment:
I want to underline that it needs to be a specialized ```PermissionDescriptor```. I will eat the redundancy for the sake of clarity for now. Let see if that still feels redundant later.